aws-mssql.yml

```diff
@@ -263,6 +263,12 @@ provision:
       Allow minor version upgrades automatically during the maintenance window.
       If `auto_minor_version_upgrade` is enabled, you must specify a major engine version.
     default: true
+  - <<: *nullable_string
```

Comment:
question: nullable means it is possible to set from string to null, but at the same time we are setting the value as `prohibit_update`, could we use string without default value?  should it have the same effect?

Reply:
good catch!